.ci/scripts/test_model.sh

```diff
@@ -50,24 +50,21 @@ prepare_artifacts_upload() {
 
 build_cmake_executor_runner() {
   echo "Building executor_runner"
-  (rm -rf ${CMAKE_OUTPUT_DIR} \
-    && mkdir ${CMAKE_OUTPUT_DIR} \
-    && cd ${CMAKE_OUTPUT_DIR} \
-    && retry cmake -DCMAKE_BUILD_TYPE=Release \
+  rm -rf ${CMAKE_OUTPUT_DIR}
+  cmake -DCMAKE_BUILD_TYPE=Debug \
```

Comment:
Please mention the move from Release to Debug in the PR summary. Consider adding a comment here explaining why we use this build mode here.

But besides that and removing `retry`, I don't see a behavior change here: it still removes the directory and generates the cmake system.